src/components/userProfileLayout/privateProfileElements/header/privateProfileHeader.tsx

```diff
@@ -20,9 +19,7 @@ const PrivateProfileHeader = ({ dollar, euro, crypto }: ICurrentRate) => {
 		<div className={styles.userProfileHeaderWrap}>
```

Comment:
can you remove src/components/currencyHeader/currencyHeader.tsx and load currency values directly in PrivateProfileHeader component?